bitcraze_crazyflie_2/LICENSE

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2024 varadVaidya
```

Comment:
@varadVaidya This should preserve the original copyright name from the ROS repo.